tests/unit/models/test_feature.py

```diff
@@ -87,6 +87,7 @@ def test_feature_model(feature_model_dict, api_object_to_id):
         "version": None,
         "definition": None,
         "catalog_id": DEFAULT_CATALOG_ID,
+        "primary_entity_ids": [],
```

Comment:
This is an example that `primary_entity_ids` is set to empty list when it does not exist in `feature_model_dict` dictionary.